hstream-connector/HStream/Connector/ClickHouse.hs

```diff
@@ -6,24 +6,23 @@ module HStream.Connector.ClickHouse
   ( clickHouseSinkConnector
   ) where
 
-import           Control.Monad                   (join, void)
+import           Control.Monad                   (void)
 import qualified Data.Aeson                      as Aeson
-import           Data.Bifunctor                  (first)
 import qualified Data.ByteString.Char8           as DBC
 import qualified Data.HashMap.Strict             as HM
 import           Data.List                       (intercalate)
 import           Data.Scientific                 (floatingOrInteger)
-import           Data.Text                       (Text)
 import qualified Data.Text                       as Text
 import qualified Database.ClickHouseDriver       as CK
 import qualified Database.ClickHouseDriver.Types as CK
+import           HStream.Connector.Converter
 import           Haxl.Core                       (Env)
 import qualified Z.IO.Logger                     as Log
 
 import           HStream.Processing.Connector    (SinkConnector (..))
 import           HStream.Processing.Type         (SinkRecord (..))
```

Comment:
Please **DO** read this: https://docs.hstream.io/development/haskell-style/#imports

Reply:
I thought only one function will be fine. My fault.